src/py/mat3ra/made/tools/analyze.py

```diff
@@ -265,13 +268,16 @@ def get_atom_indices_with_condition_on_coordinates(
 def get_nearest_neighbors_atom_indices(
     material: Material,
     coordinate: Optional[List[float]] = None,
+    tolerance: float = 0.1,
```

Comment:
Units in comments for both tolerance and cutoff. What are coordinate units - angstrom or crystal?

Reply:
tolerance -- unitless, cutoff -- angstroms. Added to Docstring

Reply:
Let's just put a link to the original implementation by PymatgenVoronoiNN for tolerance